packages/lesswrong/components/posts/usePostsUserAndCoauthors.ts

```diff
@@ -10,7 +10,7 @@ export const usePostsUserAndCoauthors = (post: PostsList|SunshinePostsList) => {
     topCommentAuthor = null;
   }
 
-  const coauthors = post.coauthors.filter(({_id}) => !postCoauthorIsPending(post, _id));
+  const coauthors = post.coauthors?.filter(({_id}) => !postCoauthorIsPending(post, _id));
   const authors = [post.user, ...coauthors].filter(
     (user): user is UsersMinimumInfo => !!user,
   );
```

Comment:
I think we'll want a `const coauthors = ... ?? []` or else the next line can error as well

Reply:
Oh, yep, that sure is true 🤦